code/modules/client/preferences_savefile.dm

```diff
@@ -244,7 +244,7 @@
 	r_eyes			= sanitize_integer(r_eyes, 0, 255, initial(r_eyes))
 	g_eyes			= sanitize_integer(g_eyes, 0, 255, initial(g_eyes))
 	b_eyes			= sanitize_integer(b_eyes, 0, 255, initial(b_eyes))
-	underwear		= sanitize_integer(underwear, 1, underwear_m.len, initial(underwear))
+	underwear		= sanitize_integer(underwear, 1, underwear_m.len | underwear_f.len, initial(underwear))
```

Comment:
I'm quite sure you don't want to be bitwise ORing the lens here. You want to be using _m.len if the gender is male and _f.len if the gender is female.